### PR TITLE
Import

### PR DESCRIPTION
**Pull request by Danble**
#### Relevant Issue
N/A

#### Summarize what changed in this PR (for developers)
Besides new changes in semantic domains and App-translations, I added manually new semantic domains to mappings because with Vite is not automated this change yet.

#### Summarize changes in this PR (for public-facing changelog)
New changes in semantic domains and App-translations
image

#### How can the changes be tested? Please also provide applicable links using preview deployments once they are available (will require editing this message once the preview deployment is ready).
https://living-dictionaries-8ov1tuxri-livingtongues.vercel.app/achi/entry/8OvJG6Fx2hmqReDunb1G

### CODE REVIEW

**Comment by Danble** [comment]
> #### Relevant Issue
> N/A
> 
> #### Summarize what changed in this PR (for developers)
> Besides new changes in semantic domains and App-translations, I added manually new semantic domains to mappings because with Vite is not automated this change yet.
> 
> #### Summarize changes in this PR (for public-facing changelog)
> New changes in semantic domains and App-translations
> image
> 
> #### How can the changes be tested? Please also provide applicable links using preview deployments once they are available (will require editing this message once the preview deployment is ready).
> https://living-dictionaries-8ov1tuxri-livingtongues.vercel.app/achi/entry/8OvJG6Fx2hmqReDunb1G

File: src/locales/pt.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
     "two_decades_experience": "Com quase duas décadas de experiência no trabalho de campo, os investigadores do Living Institute for Endagered Languages tem competências e métodos actuais para ajudar as comunidades a documentar, proteger e promover as suas línguas.",
     "help_us_launch_more": "Ajude-nos a lançar mais Dicionários Vivos",
     "with_your_help": "Com a sua ajuda poderemos continuar a melhorar o nosso software e torná-lo disponível para qualquer comunidade que pretenda criar o seu Dicionário Vivo. Doe hoje e tenha um impacto!",
-    "translators": "Tradutores"
+    "translators": "Tradutores",
+    "interns": "Estagiários."
```
Comment (jacobbowdoin) [inline src/locales/pt.json:29]: Remove period

File: src/locales/sw.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
     "two_decades_experience": "Kwa uzoefu wa kazi za uwandani wa karibu miongo miwili, watafiti wa Taasisi ya Living Tongues ya Lugha Zilizo Hatarini wana ujuzi na mbinu za kuzisaidia jamii kuhifadhi, kulinda, na kukuza lugha zao.",
     "help_us_launch_more": "Tusaidie Kuzindua Kamusi Hai Nyingi Zaidi",
     "with_your_help": "Kwa msaada wako, tunaweza kuendelea kuboresha programu yetu na kuifanya ipatikane kwa jamii yoyote inayopenda kuunda Kamusi Hai. Changia leo ili ulete mabadiliko!",
-    "translators": "Watafsiri"
+    "translators": "Watafsiri",
+    "interns": "Interns."
```
Comment (jacobbowdoin) [inline src/locales/sw.json:29]: Remove period

**Review by jacobbowdoin — APPROVED** [review]
Good to go, just remove those 3 periods

**Review by jacobbowdoin — APPROVED** [review]
Great! And I'm assuming the periods were removed from the spreadsheet and not just from the code.

**Pull request merged**